### DISCUSSION TITLE
Why are Member and User section empty?

### DISCUSSION THREAD

**Discussion by enkelmedia** [umbraco/Umbraco-CMS#17750] [Features and ideas]
Hi!

Just wanted to raise this as I think it's kind of a usability issue and also might be confusing for newcomers.

When I navigate to Member or Users the only thing I see is a empty view. I get that technically it's a blank space where someone could put a dashboard but I don't think that the empty view is great.

![image](https://github.com/user-attachments/assets/4ef6a171-cbe3-4a06-b047-07fdad7ff5f6)

![image](https://github.com/user-attachments/assets/18d39750-c55b-4854-abd5-eaec9018fdaf)

I think that there are several things that could be done here and here are some ideas

**Simple solution**
If there are no dashboards, direct to the users overview

**More ambitious**
Maybe provide real dashboards with quick actions for adding new users/members, list of latest logins etc.

What do people think?

**Comment by madsrasmussen** [comment]
Hi Marcus, 

This is a really good question. Sometimes dreams work faster than code. 😄 

I would like to introduce a "generic" dashboard concept across all sections. A dashboard should consist of "Dashboard App"-extensions that anyone can register. An app can include any information that is relevant to highlight for that specific section. For users, it could be locked-out users, pending invites, etc.

This is an attempt to generalize the many dashboard concepts we have seen over time and instead of adding yet another dashboard, then we can extend with dashboard apps (it should still be possible to add additional dashboards if needed).

In the most ideal solution, it would be possible for the user to choose which apps are relevant for them to see and organize their dashboards to fit their needs.

I worked on a POC a while ago and got pretty far with the concept, but more proceeding things took over. At some point, I need to move the code to the CMS repo and pick it up again.

Until then we could look into a solution to redirect to the first menu item 😄 

https://github.com/umbraco/Umbraco.CMS.Backoffice/compare/release/15.0...poc/user-management-dashboard

**Comment by enkelmedia** [comment]
Thanks for taking the time to answer @madsrasmussen :)

Indeed :) Dreams work faster than code! ;) Lovely quote! :)

Me and some of the community members (e.g. @glombek) made some prototyping and POC-work during CodeCabin, the idea was for this to be a new version of my "The Dashboard"-package. But at the end of the day, initial idea with the package was to just make us all aware of experience for non admins and editors and it would make a lot more sense to have something in the core.

There is a discussion here:

https://github.com/enkelmedia/TheDashboard/discussions/106

The idea that we had here was to ship some "out of the box" widgets (like the once currently in the package) and then allow anyone to build a "widget" (sounds a lot like your `dashboard app`) and then for logged in editors to be able to add widgets and adjust the look feel according to their needs.

If you think that this might be something to consider for the core we could probably focus our efforts into helping out/contributing instead of implementing this as a separate package - OR we could implement the ideas and then "move" it into the core later on. What do you think?

Probably a different discussion bust just wanted to make you aware about the fact that we've been working on a similar concept.

> "Until then we could look into a solution to redirect to the first menu item"

That sounds awesome!

**Comment by madsrasmussen** [comment]
Isn't that amazing? It sounds like we are aligned 90% of the way. 

I would love to develop an extendable dashboard foundation for "core." I think it makes perfect sense that we provide this, and it will at the same time give us the possibility to align our UI and code.

If it makes sense to you, then I am more than happy to open up for contributions/collaboration. Here is a little bit of information about where I was heading/investigating:

I am currently very inspired by how Apple/iOS is doing their widgets. The idea of a fixed grid with the ability to register dashboard apps in fixed sizes—"Small," "Medium," "Large," etc.—is appealing to me. This would give the possibility to tailor a different UX depending on which size the end user chooses to include in their dashboard. I imagine a package could register the same feature for multiple sizes but provide more detailed information the bigger the app size. 

Another place I also look for inspiration is the Homey Dashboard feature: https://homey.app/en-us/news/introducing-homey-dashboards. I think they have solved dashboards (very much like Apple) in a very nice way.

**Comment by enkelmedia** [comment]
@madsrasmussen Awesome, I think this would be a great improvement, and having it in the core would of course make it a lot more appealing to other package devs - not a lot of devs would create a widget for "The Dashboard"-package.

Like I said, the goal of the "The Dashboard" was to one day become obsolete 😊

I kind of like the "small, medium,large"-approach. My initial impression of the Homey dashboards is that the overall impression is kind of "cluttered" (maybe its just all the colors 😊), I also think that some widgets might need horizontal space (like a timeline chart) while others need more vertical space (like a list of recent changes) so it would be nice to "capture" when defining the sizes as well. 

Do you think we should move this to a discussion so that other community members can contribute ideas, or what would be a good way forward?

**Comment by Girish-kl33** [comment]
**You make a great point**—having a completely blank page can be a bit disorienting, especially for new users who aren’t yet familiar with the layout or features. **Redirecting to an existing overview when no dashboards are available sounds like a quick win, and your more ambitious idea of providing a default dashboard with quick actions and recent activity would definitely improve the initial experience.**

In one of my projects, I created a dashboard for members; one screenshot is attached below.
![image123](https://github.com/user-attachments/assets/795a7caf-fc92-4046-8d24-569a316141a5)
